lib/streamlit/runtime/scriptrunner/script_runner.py

```diff
@@ -116,6 +117,43 @@ class ScriptRunnerEvent(Enum):
 """
 
 
+def _mpa_v1(main_script_path: str):
```

Comment:
nit: I think a comment would be beneficial here explaining that the function transforms the pages folder into the MPAv2 representation